Project-Aurora/Project-Aurora/Devices/Omen/OmenDevices.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Aurora.Settings;
+
+namespace Aurora.Devices.Omen
+{
+    public class OmenDevices : Device
+    {
+        bool kbConnected = false;
+        bool peripheralConnected = false;
+
+        List<IOmenDevice> devices;
+
+        private bool isInitialized = false;
+        private readonly string devicename = "OMEN";
+
+        private readonly System.Diagnostics.Stopwatch watch = new System.Diagnostics.Stopwatch();
+        private long lastUpdateTime = 0;
+
+        public string GetDeviceDetails()
```

Comment:
Only noticed this now, please swap these methods: GetDeviceDetails and GetDeviceName.